### PR TITLE
avidemux: add qt5.15 patch.

### PR DESCRIPTION
**Pull request by fosslinux**

### CODE REVIEW

File: srcpkgs/avidemux/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'avidemux'
 pkgname=avidemux
 version=2.7.4
-revision=5
+revision=6
```
Comment (Johnnynator) [inline srcpkgs/avidemux/template:4]: Any reason why it has to be rebuild?

File: srcpkgs/avidemux/patches/qt5.15.patch [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+diff --git avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp
+index 59180d1dd..3c115a824 100644
+--- avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp
++++ avidemux/qt4/ADM_UIs/src/T_RubberControl.cpp
+@@ -12,6 +12,10 @@
+ #include <QBoxLayout>
+ #include <QSizeGrip>
+ 
++#if QT_VERSION >= QT_VERSION_CHECK(5,15,0)
```
Comment (Johnnynator) [inline srcpkgs/avidemux/patches/qt5.15.patch:18]: I suppose the patch is from upstream? Since that `#if` is totally unnecessary.

Comment (fosslinux) [reply srcpkgs/avidemux/patches/qt5.15.patch:18]: Yeah, it's a cherry pick from upstream -- I just needed it for local to build.

File: srcpkgs/avidemux/template (hunk 1 above)
Comment (fosslinux) [reply srcpkgs/avidemux/template:4]: Strictly speaking no. I can remove if that is preferable. I erred on the side of caution, I haven't noticed any problems with it though

**Pull request merged**